src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,7 +25,4 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
-
-    public void finish(CommandCall commandCall) {

```

Comment:
Yeah, that does seem somewhat unlikely!

(I didn't realise it was new for 2.0.0 - which is odd given I probably
added it!)

Tim

On 1 November 2015 at 20:55, Nigel Charman <notifications@github.com> wrote:

> In src/main/java/org/concordion/api/AbstractCommand.java
> <https://github.com/concordion/concordion/pull/138#discussion_r43579947>:
>
> > @@ -25,7 +25,4 @@ public boolean isExample() {
> >
> >      public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
> >      }
> > -
> > -    public void finish(CommandCall commandCall) {
>
> Given that this method is new in 2.0.0 and we have only published a
> SNAPSHOT, I'm not too fussed about the potential breakage.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/concordion/concordion/pull/138/files#r43579947>.
>



-- 
Tim
021 251 5593
http://www.linkedin.com/in/drtimwright